examples/asr/transcribe_speech.py

```diff
@@ -107,6 +107,13 @@ class ModelChangeConfig:
     conformer: ConformerChangeConfig = ConformerChangeConfig()
 
 
+@dataclass
+class BeamSearchConfig:
```

Comment:
to rename to `AEDBeamSearchConfig`

Reply:
@stevehuang52 move this dataclass to transformer code where beam search is implemented